core/scheduler/scheduler.go

```diff
@@ -53,6 +53,8 @@ func NewForT(t *testing.T, clock clockwork.Clock, delayFunc delayFunc, pubkeys [
 
 // New returns a new scheduler.
 func New(pubkeys []core.PubKey, eth2Cl eth2wrap.Client, builderAPI bool) (*Scheduler, error) {
+	totalValsGauge.Set(float64(len(pubkeys)))
```

Comment:
setting the metric here since it is a constant value and needs to be set only once (when scheduler is constructed)